scalameter-core/src/main/scala/org/scalameter/Measurer.scala

```diff
@@ -12,6 +16,21 @@ trait Measurer extends Serializable {
   def name: String
   def measure[T, U](context: Context, measurements: Int, setup: T => Any, tear: T => Any, regen: () => T, snippet: T => Any): Seq[Double]
   def units: String
+
+  /** Prepares a context that the setup context should be concatenated to.
+   *
```

Comment:
Could you write:

    Modifies the initial test context.
    
    This method is invoked before the `PerformanceTest` object's constructor is 
    invoked.
    The key-value pairs that the `Measurer` adds to the Context in this method are 
    visible to all the test snippets within the `PerformanceTest` class.

    Most measurers do not need to add any specific keys, so the default 
    implementation just returns the `context`.